File: C#/디자인패턴/Teamplate/Article.cs
public class Article
{
    private String title;
    private List<String> content;
    private String footer;

    public Article(string title, List<string> content, string footer)
    {
        this.title = title;
        this.content = content;
        this.footer = footer;
    }

    public String getTitle() { return title; }
    public List<String> getContent() { return content; }
    public String getFooter() { return footer; }
}

File: 디자인패턴/java 디자인 패턴/Template/Article.java
import java.util.ArrayList;

public class Article {
    private String title;
    private ArrayList<String> content;
    private String footer;

    public Article(String title, ArrayList<String> content, String footer) {
        this.title = title;
        this.content = content;
        this.footer = footer;
    }

    public String getTitle() {
        return title;
    }
    
    public ArrayList<String> getcontent() {
        return content;
    }

    public String getfooter() {
        return footer;
    }
}